sql/core/src/test/scala/org/apache/spark/sql/MathFunctionsSuite.scala

```diff
@@ -173,6 +173,14 @@ class MathFunctionsSuite extends QueryTest with SharedSQLContext {
     checkAnswer(
       sql("SELECT ceiling(0), ceiling(1), ceiling(1.5)"),
       Row(0L, 1L, 2L))
+    
+    checkAnswer(
+      sql("SELECT ceil(1234567890123456), ceil(12345678901234567)"),
+      Row(1234567890123456L, 12345678901234567L))
+
+    checkAnswer(
+      sql("SELECT ceiling(1234567890123456), ceiling(12345678901234567)"),
+      Row(1234567890123456L, 12345678901234567L))
```

Comment:
ok, add new tests to the end of operators.sql.
please review it again.
thanks.